### ISSUE TITLE
Terminal app should not have a custom behaviour for keyboard

### ISSUE THREAD

**Issue by ydirson** [radekp/qtmoko#103]
Current keyboard behaviour in Terminal has many problems:
- different from the rest, it is too unpredictible
- long press on POWER, then Cancel, leaves a terminal app without a keyboard, and no obvious way to reactivate it

Why not simply use the same behaviour as the rest of the system (show keyboard when focus is in input widget) ?

**Comment by radekp** [comment]
Yes it should behave the same as rest of system. I think it's different now, because the terminal was written for old Qtopia. It would be nice if you could find how to make it behave sane.

**Comment by radekp** [comment]
I believe that with new svg keyboard and the keyboard button in bottom bar this is no longer problem.

**Comment by edorfaus** [comment]
This is still a problem in v57 with the SVG keyboard, because the keyboard button in the bottom bar does not show up in Terminal. I'm guessing that it not showing up is caused by the same reason as why the keyboard behaved differently before, but I haven't had a chance to look into it.

The Options menu includes a keyboard submenu with a hide/show entry, which appears to work the same way as the keyboard button, so there's a workaround for it being missing.
